lib/model.js

```diff
@@ -2892,6 +2892,7 @@ class Model {
   * @return {Promise<this>}
   */
   static increment(fields, options) {
+    this._injectScope(options);
```

Comment:
Having this before `_optionsMustContainWhere` makes sense to me, but if there's a reason to inject scopes after that assertion, we can move it down.

Reply:
I think it fits fine here